cdap-support-bundle/src/test/java/io/cdap/cdap/support/handlers/SupportBundleHttpHandlerTest.java

```diff
@@ -73,23 +62,23 @@ public void testCreateSupportBundleWithValidNamespace() throws Exception {
    */
   private String requestBundle(Map<String, String> params) throws IOException {
     DiscoveryServiceClient discoveryServiceClient = getInjector().getInstance(DiscoveryServiceClient.class);
-    Discoverable discoverable = new RandomEndpointStrategy(
-      () -> discoveryServiceClient.discover(Constants.Service.SUPPORT_BUNDLE_SERVICE)).pick(5, TimeUnit.SECONDS);
+    Discoverable discoverable =
+      new RandomEndpointStrategy(() -> discoveryServiceClient.discover(Constants.Service.SUPPORT_BUNDLE_SERVICE)).pick(
+        5, TimeUnit.SECONDS);
 
     Assert.assertNotNull("No service for support bundle", discoverable);
 
     StringBuilder queryBuilder = new StringBuilder();
     String sep = "?";
     for (Map.Entry<String, String> entry : params.entrySet()) {
-      queryBuilder
-        .append(sep)
+      queryBuilder.append(sep)
         .append(URLEncoder.encode(entry.getKey(), "UTF-8"))
         .append("=")
         .append(URLEncoder.encode(entry.getValue(), "UTF-8"));
       sep = "&";
     }
 
-    String path = String.format("%s/support/bundle%s", Constants.Gateway.API_VERSION_3, queryBuilder);
+    String path = String.format("%s/support/bundles%s", Constants.Gateway.API_VERSION_3, queryBuilder);
```

Comment:
Open a JIRA to extend / add test for downloading bundle.

Reply:
I created the jira ticket https://cdap.atlassian.net/browse/CDAP-18564

Reply:
Seems like not a correct JIRA??